module/Application/src/Application/Controller/UsersSecondaryEmailsController.php

```diff
@@ -73,6 +74,16 @@ public function replaceList($emails)
             return $this->response;
         }
 
+        foreach ($emails as $email) {
```

Comment:
Ho un dubbio sulla logica: qui stiamo mandando una lista di email secondarie da aggiornare, è necessario verificare che quelle email non siano usate da qualcun altro (a parte te stesso) sia che si tratti di email primaria che secondaria, corretto?

```
Esempio 
Utente 1: u1@example.com, [doppia@example.com, altra@example.com]
Utente 2: u2@example.com, [boh@example.com]
```
siamo loggati come `Utente 1` e vogliamo aggiornare la lista di email secondarie. 

Se inserisco
```
[u2@example.com, doppia@example.com, altra@example.com]
```
devo avere un errore perché `u2@example.com` è già usata da `Utente 2`

Se inserisco 
```
[boh@example.com, doppia@example.com, altra@example.com]
```
devo avere un errore perché `boh@example.com` è già usata da `Utente 2`

**Con la query attuale viene controllata solo l'email primaria.** 

L'inghippo ulteriore è che è necessario escludere l'utente per cui si sta cercando di cambiare l'email, altrimenti si ottiene errore

```
Esempio
Utente 1: u1@example.com, [doppia@example.com, altra@example.com]
Utente 2: u2@example.com, [boh@example.com]
```

siamo loggati come Utente 1 e vogliamo aggiornare la lista di email secondarie. 

Se invio 
`[doppia@example.com, altra@example.com]` devo verificare che quelle email non siano presenti in nessun altro utente **a parte Utente 1 (sono già le sue)**

Fila il ragionamento? Ho perso dei pezzi?